invenio_accounts/api.py

```diff
@@ -8,25 +8,60 @@
 
 """API objects for Invenio Accounts."""
 
-from collections import defaultdict
+
+class Session:
+    """Session object for DB Users change history."""
+
+    def __init__(self):
+        self.updated_users = []
+        self.updated_roles = []
+        self.deleted_users = []
+        self.deleted_roles = []
+        self.indexed = False
```

Comment:
where are `indexed` and `invalidated_cache` accessed? cant find on this pr might be in another linked one?

Reply:
Discussed with @jrcastro2 IVL, since signals are sync and the cleanup is done by `clear_dirty_sets` can this two flags be removed? are they used?